### ISSUE TITLE
pkglint -e doesn't error package, what doesn't has LICENSE

### ISSUE THREAD

**Issue by miwarin** [rillig/pkglint#1]
Hi.
'pkglint -e' no LICENSE error. in today pkgsrc-wip package:

- py-visionegg-demos
- py-visionegg-contrib

these package doesn't has LICENSE. buf pkglint -e no error.

```
% pkglint -e wip/py-visionegg-demos
ERROR: py-visionegg-demos/../../wip/py-visionegg/buildlink3.mk:18: Cannot read "py-visionegg-demos/../../mk/opengl.buildlink3.mk".
1 error and 0 warnings found.
```

```
% pkglint -e wip/py-visionegg-contrib 
ERROR: py-visionegg-contrib/../../wip/py-visionegg/buildlink3.mk:18: Cannot read "py-visionegg-contrib/../../mk/opengl.buildlink3.mk".
1 error and 0 warnings found.
```

**Comment by rillig** [comment]
Thanks for the report. My first guess was that the package Makefile was too complicated, so that pkglint couldn't decide on whether there _may_ be a license defined or not. But this is not the case here.

In this case, pkglint tries to read the package Makefile, which involves reading all included files, recursively. That fails because of the one file that cannot be found (see the error message), so pkglint thinks that the whole package Makefile could not be read. And because of that, all further checks are skipped.

This is definitely surprising, so I need to fix it.

When running pkglint on the whole pkgsrc tree, including wip, this error message appears exactly 2 times, and you know where. Therefore I think the proper fix is to ignore missing files if they are guarded by an `if exists`. That would fix both error messages and also sounds sane in general.

**Comment by rillig** [comment]
Fixed in version 5.5.10, which has just been updated in pkgsrc-current.
